apps/system/test/unit/activities_test.js

```diff
@@ -82,16 +79,38 @@ suite('system/Activities', function() {
     });
 
     test('hides actionMenu on appopended if it exists', function() {
-      var stub = this.sinon.stub(ActionMenu.prototype, 'hide');
+      this.sinon.stub(ActionMenu.prototype, 'hide');
       subject.actionMenu = new ActionMenu();
       subject.handleEvent({type: 'appopened'});
-      assert.ok(stub.calledOnce);
+      assert.ok(ActionMenu.prototype.hide.calledOnce);
+    });
+
+    test('checks default actions when new app installed', function() {
+      this.sinon.stub(subject, '_onNewAppInstalled');
+
+      subject.handleEvent({
+        type: 'applicationinstall',
+        detail: {
+          application: {}
+        }
+      });
```

Comment:
this test is barely testing anything... we can remove it.
(We're covered by the other test below where we really dispatch an `applicationinstall` event)